### ISSUE TITLE
Does it support Dependency Parsing?

### ISSUE THREAD

**Issue by oney** [NaturalNode/natural#487]
I notice this project supports POS tagging but not Dependency Parsing.
https://universaldependencies.org/
https://nlp.stanford.edu/software/nndep.html
https://en.wikipedia.org/wiki/Dependency_grammar
https://www.youtube.com/watch?v=PVShkZgXznc

Am I right?

**Comment by Hugo-ter-Doest** [comment]
Yes, you are right. I have some plans to add CFG parsing and unification-based parsing.

Feel free to make a PR with a module for dependency parsing!
